javascripts/discourse/components/top-contributors.js

```diff
@@ -4,17 +4,23 @@ import { tracked } from "@glimmer/tracking";
 
 export default class TopContributors extends Component {
   @tracked topContributors = null;
+  order = null;
+  viewAllUrl = null;
 
   constructor() {
     super(...arguments);
-
     const count = this.args?.params?.count || 5;
+    this.order = this.args?.params?.order || "likes_received";
+    const period = this.args?.params?.period || "yearly";
```

Comment:
`this.args` is never null/undefined so those `?.` immediately following it aren't necessary.